python/docs/source/examples/time_centric.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
Edited this in Jupyter so we can iterate easily. A few of the cells have added "tags" which let us control whether their output shows up in the page or not, etc.